resources/home/dnanexus/make_workbook.py

```
@@ -745,11 +779,31 @@ def create_additional_analysis_page(self):
+
+            # Get chr, pos, ref, alt for use in MT GEL tier lookup
+            coords = snv.get("variantCoordinates", {})
+            chr_ = coords.get("chromosome")
+            pos = coords.get("position")
+            ref = coords.get("reference")
+            alt = coords.get("alternate")
+
+
+            # If MT GEL variant not found, skip to next variant
+            if None in [chr_, pos, ref, alt]:
+                raise ValueError(f"Exomiser SNV missing required coordinates: (chromosome/position/ref/alt)")
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)